app/src/main/java/org/simple/clinic/widgets/PatientSearchResultItemView.kt

```diff
@@ -52,11 +52,26 @@ class PatientSearchResultItemView(
     renderPatientNameAgeAndGender(name)
     renderPatientAddress(model.address)
     renderPatientDateOfBirth(model.dateOfBirth)
-    renderPatientPhoneNumber(model.phoneNumber)
+    val number = highlightPatientPhoneNumber(searchQuery, model)
```

Comment:
We can move this inside the `renderPatientPhoneNumber` method

Reply:
Improved in the later commit, please check.